app/src/main/res/drawable/ic_edit_white.xml

```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Keep only one icon for edit, the white one. If you need it on another color, use the iconTint property